packages/agent/src/actor.ts

```diff
@@ -49,6 +49,11 @@ export interface CallConfig {
    * The effective canister ID. This should almost always be ignored.
    */
   effectiveCanisterId?: Principal;
+
+  /**
+   * The nonce to use for this call. This is used to prevent replay attacks.
+   */
+  nonce?: Uint8Array;
```

Comment:
This allows overriding the nonce when making a call using an actor. I don't normally like to change the public interface for tests, but in this case I couldn't think of a reason why we'd let you set the nonce when calling through the agent directly, but not through the actor.